bpf/lib/icmp6.h

```diff
@@ -30,8 +30,10 @@
 #define ICMP6_MULT_RA_MSG_TYPE		151
 #define ICMP6_MULT_RT_MSG_TYPE		153
 
-/* If no specific action is specified, drop unknown neighbour solicitation
- * messages.
+#define SKIP_HOST_FIREWALL	-1
```

Comment:
Maybe use a lower value? Although we are unlikely to ever use it, `TC_ACT_UNSPEC` is `-1`.